### DISCUSSION TITLE
Toggling default cursor

### DISCUSSION THREAD

**Discussion by iton0** [AlmasB/FXGL#1379] [Q&A] [answered]
I am wanting to have the default cursor visible when in the main menu or game menu but hidden when I resume the game for example. Is it possible to implement it as such?

**Comment by AlmasB** [comment, marked as answer]
Hi, when [customising menus](https://github.com/AlmasB/FXGL/wiki/Customizing-Menus-%28FXGL-11%29), you can retrieve content root of the relevant scene (e.g. main menu / game menu) and set its cursor to default.

For hiding it in the game, you can call: `getGameScene().setCursorInvisible()`
